std/traits.d

```diff
@@ -5620,6 +5620,13 @@ enum bool isMutable(T) = !is(T == const) && !is(T == immutable) && !is(T == inou
  * Returns true if T is an instance of the template S.
  */
 enum bool isInstanceOf(alias S, T) = is(T == S!Args, Args...);
+/// ditto
+template isInstanceOf(alias S, alias T)
+{
+    enum impl(alias T : S!Args, Args...) = true;
+    enum impl(alias T) = false;
+    enum isInstanceOf = impl!T;
+}
```

Comment:
I would have written this as

```d
enum bool isInstanceOf(alias S, alias T : S!Args, Args...) = true;
enum bool isInstanceOf(alias S, alias T) = false;
```

But: https://issues.dlang.org/show_bug.cgi?id=16403